apps/prairielearn/src/schemas/schemas/infoAssessment.ts

```diff
@@ -0,0 +1,371 @@
+import { z, type ZodSchema } from 'zod';
+
+import { CommentSchema } from './comment.js';
+
+function uniqueArray<T extends ZodSchema>(schema: T) {
+  return z.array(schema).refine((items) => new Set(items).size === items.length, {
+    message: 'All items must be unique, no duplicate values allowed',
+  });
+}
+
+const GroupRoleSchema = z
+  .object({
+    name: z.string().describe("The group role's name (i.e. Manager, Reflector, Recorder)."),
+    minimum: z
+      .number()
+      .describe('The minimum number of users that should be in this role in a group.')
+      .optional(),
+    maximum: z
+      .number()
+      .describe('The maximum number of users that should be in this role in a group.')
+      .optional(),
+    canAssignRoles: z
+      .boolean()
+      .describe("Whether users with this role can assign other users' group roles.")
+      .optional(),
+  })
+  .describe(
+    'A custom role for use in group assessments that allows control over certain permissions.',
+  );
+
+export const AsssessmentAccessRuleSchema = z
```

Comment:
Too many esses here (`Asssessment` -> `Assessment`)

Reply:
Resolved in 8f55a7884fa6d7fc377d041e67d4eb87a2988b73.